packages/drawer/src/mwc-drawer.ts

```diff
@@ -77,13 +85,12 @@ export class Drawer extends BaseElement {
       // TODO(sorvell): Implement list focusing integration.
       focusActiveNavigationItem: () => {
       },
-      // TODO(sorvell): integrate focus trapping.
-      trapFocus: () => {},
-      releaseFocus: () => {},
+      trapFocus: () => {
+        document.$blockingElements.push(this);
```

Comment:
Ideally, `document.$blockingElements.remove(this);` would be in `releaseFocus`. However, this doesn't work because material-components-web calls releaseFocus and restoreFocus out of order in the dismissible adapter. Opened https://github.com/material-components/material-components-web/issues/4268 as a result of this discovery.